src/frontend/scss/objects/_editor.scss

```diff
@@ -0,0 +1,55 @@
+.ashley-editor-widget {
+  border: 1px solid #ced4da;
```

Comment:
Where does those colors come from? Shouldn't we use variables instead?

Reply:
Sorry for the delay, is it better now ? :slightly_smiling_face:

Reply:
![](https://media2.giphy.com/media/3ohzdIuqJoo8QdKlnW/giphy-downsized-medium.gif)